Source/com/drew/metadata/mp4/boxes/ItemListBox.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2002-2019 Drew Noakes and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ *
+ * More information about this project is available at:
+ *
+ *    https://drewnoakes.com/code/exif/
+ *    https://github.com/drewnoakes/metadata-extractor
+ */
+package com.drew.metadata.mp4.boxes;
+
+import com.drew.lang.SequentialReader;
+import com.drew.metadata.mp4.media.Mp4MetaBoxDirectory;
+
+import java.io.EOFException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ItemListBox extends Box
+{
+    private class ItemListEntry
+    {
+        private int tag;
+        private String val;
+
+        ItemListEntry(int tag) {
+            this.tag = tag;
+            this.val = null;
+        }
+
+        int getTag() {
+            return this.tag;
+        }
+
+        void setVal(String val) {
+            this.val = val;
+        }
+
+        String getVal() {
+            return this.val;
+        }
+    }
+
+    public static Map<String, ItemListEntry> itemMap;
+
+    {
+        itemMap = new HashMap<String, ItemListEntry>();
+
+        itemMap.put("�alb", new ItemListEntry(Mp4MetaBoxDirectory.TAG_ALBUM));
+        itemMap.put("aART", new ItemListEntry(Mp4MetaBoxDirectory.TAG_ALBUM_ARTIST));
```

Comment:
Maps the item list keys to their corresponding directory key (and value if set later)